### ISSUE TITLE
Native video embeds

### ISSUE THREAD

**Issue by tommoor** [outline/outline#3130]
After #859 all the pieces are in place to support this, we just need to add an inline player for uploaded videos.

**Comment by nvolker** [comment]
Awesome! Looking forward to this one 🥇

**Comment by Limezy** [comment]
Very good news. I would also be very glad if you include an audio player 👍

**Comment by Limezy** [comment]
I don't know if it would be doable with the current software structure, but I would dream to be able to : 
* Make internal link to a timestamp of the video / audio (so that when clicking it would start the player at a precise point)
* Write some comments pinned to a timestamp of the video / audio (just like Loom app for instance)

**Comment by nikolailehbrink** [comment]
Hi, I've been wishing for this feature for some time but see that it keeps getting pushed back. Can it be implemented soon?

**Comment by actus-shiekho** [comment]
Any ideas if this will be tackled any time soon?

**Comment by shanyan-wcx** [comment]
Any updates?

**Comment by StarWhiz** [comment]
This would be very nice to have since in Confluence you could drag and drop videos into a page and it would be playable! No need for external video hosts like youtube vimeo!

**Comment by briannezhad** [comment]
Is this available?
